<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplachScreen</class>
 <widget class="QMainWindow" name="SplachScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QFrame" name="dropShadowFrame">
      <property name="styleSheet">
       <string notr="true">QFrame{
	
	background-color: rgb(56, 58, 89);
	color: rgb(220, 220, 220);
	border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_title">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>80</y>
         <width>651</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">/*color: rgb(229, 152, 155);*/
color: rgb(112, 160, 175);
color: rgb(191, 204, 181);</string>
       </property>
       <property name="text">
        <string>&lt;strong&gt;Plagiat&lt;/strong&gt;Scanner</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_description">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>160</y>
         <width>651</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">/*color: rgb(181, 131, 141);*/
color: rgb(229, 152, 155);
color: rgb(191, 204, 181);
color: rgb(112, 160, 175);

background: none;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Plagiat&lt;/span&gt;Scanner&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>260</y>
         <width>581</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar{


	background-color: rgb(85, 88, 135);
	color: rgb(200, 200, 200);
	border-style: none;
	border-radius: 10px;
	text-align: center;
	
}

QProgressBar::chunk{
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:1, y1:0.494, x2:0, y2:0.506, stop:0 rgba(159, 105, 108, 255), stop:1 rgba(229, 152, 155, 255));
	background-color: qlineargradient(spread:pad, x1:1, y1:0.494, x2:0, y2:0.506, stop:0 rgba(112, 160, 175, 255), stop:1 rgba(229, 152, 155, 255));
	background-color: qlineargradient(spread:pad, x1:1, y1:0.494, x2:0, y2:0.506, stop:0 rgba(112, 160, 175, 255), stop:1 rgba(191, 204, 181, 255));
	background-color: qlineargradient(spread:pad, x1:1, y1:0.494, x2:0, y2:0.506, stop:0 rgba(94, 136, 148, 255), stop:1 rgba(191, 204, 181, 255));

}</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_loading">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>280</y>
         <width>651</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(181, 131, 141);
color: rgb(191, 204, 181);
color: rgb(112, 160, 175);
background: none;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Chargement...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
